### DISCUSSION TITLE
Open boundary condition with tangential velocity component

### DISCUSSION THREAD

**Discussion by mpachera** [firemodels/fds#12337] [Q&A]
Dear,

Is there any way to impose a tangential velocity component, as explained in section 10.1.4, on an open boundary condition? I have a case with tangential wind and I placed an opening at the top, so I would also like to keep the tangential velocity component near the boundary condition. 
![image](https://github.com/firemodels/fds/assets/15084725/6056edcb-13e4-4ecb-8c6e-60545e22d59f)

With a simple open boundary condition I noticed that the gas flows straight into the opening rather than travelling along the domain. I know I can further extend the domain to limit this effect near the zone of interest, but I would like to find a more elegant solution. 
![image](https://github.com/firemodels/fds/assets/15084725/1460a2b1-fc7f-4123-998e-49bfce4dbb8c)

Thanks in advance.

Kind regards,

Matteo

**Comment by mcgratta** [comment]
Have you tried to use a `WIND` line with `SPEED`? Or you could also try `DYNAMIC_PRESSURE` on the `VENT` line with `SURF_ID='OPEN'`

**Reply by mpachera** [reply]
I couldn't use wind and speed inputs because I don't have an uniform wind velocity around my domain. I took the results of a wind analysis as boundary conditions for the fire analysis so both the velocity pressure vary locally.

**Comment by mcgratta** [comment]
I do not understand what you are trying to do. Can you create a very simple example?

**Comment by mpachera** [comment]
The line should look like this: 
&VENT ID='Vent ZMAX', SURF_ID='OPEN', VEL_T=-5,0,  XB=-2.0,2.0,-2.0,2.0,2.8,2.8/ 
The normal component of the velocity through the vent is calculated based on the pressure, but the tangential component is imposed.
